### ISSUE TITLE
[BUG] Error opening GeoIP DB

### ISSUE THREAD

**Issue by jonathantjm** [bunkerity/bunkerized-nginx#139]
**Description**
When running the bunkerized nginx docker container, an error occurs when opening the GeoIP DB.

**How to reproduce**
Run docker command with country in WHITELIST_COUNTRY or BLACKLIST_COUNTRY:

`docker run -p 80:8080 -v ~/Desktop/folder:/www:ro -e WHITELIST_COUNTRY=US bunkerity/bunkerized-nginx`

The `latest` image was used, as of time of writing.

**Logs**
The logs generated by bunkerized-nginx. 

```
[*] Starting bunkerized-nginx ...
[*] Configuring bunkerized-nginx ...
[*] Generation done !
[*] Downloading bad user-agent list (in background) ...
[*] Downloading bad referrer list (in background) ...
[*] Downloading tor exit nodes list (in background) ...
[*] Downloading proxies list (in background) ...
[*] Downloading abusers list (in background) ...
[*] Running nginx ...
2021/06/08 15:02:39 [emerg] 119#119: MMDB_open("/etc/nginx/geoip.mmdb") failed - Error opening the specified MaxMind DB file in /etc/nginx/geoip.conf:1
nginx: [emerg] MMDB_open("/etc/nginx/geoip.mmdb") failed - Error opening the specified MaxMind DB file in /etc/nginx/geoip.conf:1
[*] bunkerized-nginx stopped
```

**Comment by bunkerity** [comment]
Hello @jonathantjm,

Thanks for your report. I will investigate into this when I have time.

**Comment by bunkerity** [comment]
Hello @jonathantjm,

I've added a fix into the dev branch. Let's close the issue when it's integrated in the next version. Thanks again !

**Comment by bunkerity** [comment]
Hey @jonathantjm,

Fix has been integrated into the v1.2.7. Feel free to open a new issue if needed.
